### PR TITLE
Posture SDK

### PR DESCRIPTION
**Pull request by jasoncalalang**

### CODE REVIEW

File: posturemanagementv1/posture_management_v1_integration_test.go [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+			if authUrl == "" {
+				Skip("Unable to load auth service URL configuration property, skipping tests")
+			}
+
+			apiKey = config["IAM"]
+			if apiKey == "" {
+				Skip("Unable to load IAM configuration property, skipping tests")
+			}
+
+			credentialString = "{\"credential_name\":\"SDK CREDENTIALS PIPELINE TEST\",\"credential_description\":\"DEMO CREDENTIALS\",\"purpose\":\"discovery_fact_collection_remediation\",\"enabled\":true,\"credential_group\":{},\"credential_type\":\"ibm_cloud\",\"ibm_cloud\":{\"ibm_api_key\":\"lkjb1rsBsqGKyPBXL7YRwmd-JUl9fI38Ntym0RarZGD-\"}}"
```
Comment (ashishth09) [inline posturemanagementv1/posture_management_v1_integration_test.go:96]: API key should be read from ENV or a file

**Review by ashishth09 — CHANGES_REQUESTED** [review]

File: posturemanagementv1/posture_management_v1_integration_test.go (hunk 1 above)
Comment (gary1998) [reply posturemanagementv1/posture_management_v1_integration_test.go:96]: you can read it from a ENV var and set it up in CI ... i wonder why didn't you receive ServiceNow ticket for this 😅

**Review by gary1998 — CHANGES_REQUESTED** [review]

File: posturemanagementv1/posture_management_v1_integration_test.go (hunk 1 above)
Comment (jasoncalalang) [reply posturemanagementv1/posture_management_v1_integration_test.go]: > API key should be read from ENV or a file

accepted and updated

**Comment by gary1998** [comment]
`notificationsv1` service ITs are failing... Could you please replace `https://webhook.site/136fe1e2-3c3f-4bff-925f-391fbb202546` to `https://us-south.functions.appdomain.cloud/api/v1/web/e6b54af6-ab44-4149-a8e4-e906dcc58136/default/sdk-integration` in `https://github.com/IBM/scc-go-sdk/blob/main/notificationsv1/notifications_v1_integration_test.go` file.

File: notificationsv1/notifications_v1_integration_test.go [hunk 2]
```diff
@@ -130,7 +130,7 @@ var _ = Describe(`NotificationsV1 Integration Tests`, func() {
 				AccountID:   &accountID,
 				Name:        core.StringPtr(fmt.Sprintf("%s-%s", testString, identifier)),
 				Type:        core.StringPtr("Webhook"),
-				Endpoint:    core.StringPtr("https://us-south.functions.appdomain.cloud/api/v1/web/e6b54af6-ab44-4149-a8e4-e906dcc58136/default/sdk-integration.json"),
+				Endpoint:    core.StringPtr("https://us-south.secadvisor.cloud.ibm.com/findings/v1/api-docs"),
```
Comment (ashishth09) [inline notificationsv1/notifications_v1_integration_test.go:133]: The original is right.

**Review by ashishth09 — CHANGES_REQUESTED** [review]
The previous webhook url was correct. Let's keep that one as it is

File: notificationsv1/notifications_v1_integration_test.go (hunk 2 above)
Comment (jasoncalalang) [reply notificationsv1/notifications_v1_integration_test.go]: accepted and now reverted

File: notificationsv1/notifications_v1_integration_test.go [hunk 3]
```diff
@@ -184,7 +184,7 @@ var _ = Describe(`NotificationsV1 Integration Tests`, func() {
 				ChannelID:   &channelID,
 				Name:        core.StringPtr(fmt.Sprintf("%s-%s", testString, identifier)),
 				Type:        core.StringPtr("Webhook"),
-				Endpoint:    core.StringPtr("https://us-south.functions.appdomain.cloud/api/v1/web/e6b54af6-ab44-4149-a8e4-e906dcc58136/default/sdk-integration.json"),
+				Endpoint:    core.StringPtr("https://us-south.functions.appdomain.cloud/api/v1/web/e6b54af6-ab44-4149-a8e4-e906dcc58136/default/sdk-integration.jso"),
```
Comment (ashishth09) [inline notificationsv1/notifications_v1_integration_test.go:187]: Its a typo,  missing an "n"

Comment (jasoncalalang) [reply notificationsv1/notifications_v1_integration_test.go:187]: thanks, will fix that

Comment (jasoncalalang) [reply notificationsv1/notifications_v1_integration_test.go]: > Its a typo, missing an "n"

accepted and updated

**Pull request merged**